x-pack/plugin/eql/src/test/resources/test_queries_unsupported.toml

```diff
@@ -0,0 +1,1300 @@
+# This file is populated with currently unsupported queries.
+# Serves as a blacklist, until our implementation starts supporting a specific query
+# This file is expected to become empty once the feature parity is reached with the
+# official EQL implementation
+
+[[queries]]
+query = 'process where serial_event_id < 4'
```

Comment:
this one is supported

Reply:
yeah, but not hooked up to the query engine yet. Once it is, we can validate the results and update the .toml files.

Reply:
expecting the test_queries_unsupported.toml file to become empty and eventually dropped when we reach the feature parity

Reply:
oh, I didn't realize that. in that case, this file is missing the first query from test_queries.toml. i just looked at the top and saw the discrepancy and reached the wrong conclusion
```toml
[[queries]]
query = 'process where serial_event_id = 1'
expected_event_ids  = [1]
```

Reply:
Added the comment into the file per @rw-access recommendation.